spec/features/hiring_staff_can_edit_a_draft_vacancy_spec.rb

```diff
@@ -33,7 +33,7 @@
       fill_in_pay_package_form_fields(draft_vacancy)
       click_on I18n.t('buttons.save_and_continue')
 
-      expect(page).to have_content(I18n.t('jobs.current_step', step: 3, total: 5))
+      expect(page).to have_content(I18n.t('jobs.current_step', step: 3, total: 6))
```

Comment:
Can we store `6` somewhere central so it doesn't have to be ceremonially changed in six places every day?

Reply:
Or dynamically get it somehow?

Reply:
Same comment for `spec/features/hiring_staff_can_publish_a_vacancy_spec.rb`

Reply:
If it's a constant that can be accessed by the view too, what's the best place for that? I don't know.

Reply:
Or maybe _tests_ shouldn't use constants, so that if the constant is wrong we get to find out by running the test. Sorry for all the thinking out loud here.